src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int LIMIT = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(LIMIT));
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/156338484/fa369d89-de26-4533-89f2-3bd4f6a5dd64)

Reply:
Not fixed comment, make Random and ColorSupplier class-level variable